guides/v2.1/frontend-dev-guide/layouts/layout-types.md

```diff
@@ -86,6 +86,31 @@ Sample page layout declaration file: `<Magento_Theme_module_dir>/view/frontend/l
 </page_layouts>
 ```
 
+Use the `layout` attribute in the `page` section to define a layout type for the page. The following example shows how to use the `3 columns` page layout type for the [Wish List Sharing](https://github.com/magento/magento2/blob/2.3.1/app/code/Magento/Wishlist/view/frontend/layout/wishlist_index_share.xml#L8) page:
+
+```xml
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="3columns" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <update handle="customer_account"/>
+    <body>
+        <referenceContainer name="content">
+            <block class="Magento\Wishlist\Block\Customer\Sharing" name="wishlist.sharing" template="Magento_Wishlist::sharing.phtml" cacheable="false"/>
+        </referenceContainer>
+    </body>
+</page>
+```
+
+![Wish List Sharing. 3 columns layout page type]({{ page.baseurl }}/frontend-dev-guide/images/version.png)
```

Comment:
This only works if you add symlinks from 2.3 and 2.2 to the 2.1 image file you added.

Reply:
I used a wrong file name. Could you please run tests again?

Thank you!

Reply:
Ah, yes. You're right. I didn't look closely enough at the errors. I'll run again soon.

Reply:
> This only works if you add symlinks from 2.3 and 2.2 to the 2.1 image file you added.
Also, I've added symlinks ;)